pymongo/collection.py

```diff
@@ -970,6 +979,7 @@ def replace_one(
                 session=session,
                 let=let,
                 comment=comment,
+                operation="replace",
```

Comment:
Does this spec require this to be "replace" or should it instead be "update" (the command name).

Reply:
It should be `update`, good catch.